app/bulk_resource/views.py

```diff
@@ -599,6 +601,8 @@ def save_csv():
         if csv_storage.action == 'reset':
             OptionAssociation.query.delete()
             TextAssociation.query.delete()
+            Suggestion.query.delete()
```

Comment:
is there a reason we delete all the suggestions? could imagine a use case where we reset the DB but still want to see all the suggestions (not resource-specific ones but the general suggestions?)